### PR TITLE
Add escape quotes in html attributes

### PR DESCRIPTION
**Pull request by maoznir**

### CODE REVIEW

File: src/tags/htmltag.js [hunk 1]
```diff
@@ -240,4 +240,13 @@ function toAttribute(key, value) {
   }
 }
 
+function escapeQuotes(obj){
+  Object.keys(obj).forEach(key=>{
+    if (typeof  obj[key] === "string") {
```
Comment (asisayag2) [inline src/tags/htmltag.js]: @maoznir , why not using Utils.isString function? 
https://github.com/cloudinary/cloudinary_js/blob/7b8492869f9e3cef1b09f1f16437796beca0f580/src/util/jquery.js#L111

File: src/tags/htmltag.js [hunk 2]
```diff
@@ -240,4 +240,13 @@ function toAttribute(key, value) {
   }
 }
 
+function escapeQuotes(obj){
+  Object.keys(obj).forEach(key=>{
```
Comment (asisayag2) [inline src/tags/htmltag.js]: @maoznir why not using map?

Comment (maoznir) [reply src/tags/htmltag.js]: done

File: src/tags/htmltag.js (hunk 1 above)
Comment (maoznir) [reply src/tags/htmltag.js]: done

File: src/tags/htmltag.js [hunk 3]
```diff
@@ -240,4 +240,11 @@ function toAttribute(key, value) {
   }
 }
 
+function escapeQuotes(obj) {
+    return Object.keys(obj).map(key => utils.isString(obj[key]) ?
```
Comment (tocker) [inline src/tags/htmltag.js]: Missing import utils

Comment (tocker) [inline src/tags/htmltag.js]: The function accepts an object of key=>value but returns an array of value[]. You need to return an object with key=>value.

File: test/spec/tagspec.js [hunk 4]
```diff
@@ -86,6 +86,14 @@ describe("Cloudinary.ImageTag", function() {
     tag = new cloudinary.ImageTag('image_id', config).toHtml();
     return expect(tag).toBe(`<img src="${DEFAULT_UPLOAD_PATH}image_id">`);
   });
+  it("should escape quotes in html attributes", function() {
+    var tag;
+    tag = new cloudinary.ImageTag('image_id', {
+      'cloud_name': 'test123',
```
Comment (tocker) [inline test/spec/tagspec.js]: Use the `config` variable.

Comment (maoznir) [reply test/spec/tagspec.js]: Used it with Object.assign to add "alt" param

File: src/tags/htmltag.js (hunk 3 above)
Comment (maoznir) [reply src/tags/htmltag.js]: done - reverted back to forEach

Comment (maoznir) [reply src/tags/htmltag.js]: done

**Pull request merged**